sktime/transformations/series/theta.py

```diff
@@ -18,13 +18,39 @@
 class ThetaLinesTransformer(_SeriesToSeriesTransformer):
     """Decompose the original data into two or more Theta-lines.
 
+    Implementation of decomposition for Theta-method [1]_ as described in [2]_.
+
+    Overview: Input :term:`univariate series <Univariate time series>` of length
+    "n" and ThetaLinesTransformer modifies the local curvature of the time series
+    using Theta-coefficients passed through the parameter `theta`.
+
+    The pimary parameter:
```

Comment:
Instead of making this a bullet point, I'd rephrase it into a sentence, it looks a bit strange on the website.